modules/ROOT/pages/munit-coverage-report.adoc

```diff
@@ -14,7 +14,7 @@ MUnit Coverage provides metrics for:
 * Flow coverage: Refers to any of the following components: Flows, Sub-flows, and Batch jobs.
 
 [NOTE]
-Coverage report runs only for EE edition runtimes
+Coverage report runs only for EE edition runtimes. If your application has a `requiredProduct` of a Mule CE runtime you will see the following message: *Coverage is a EE only feature and you've selected to run over CE*. To run against an EE runtime see xref:to-configure-munit-maven-plugin-maven.adoc#to-specify-the-runtime-product[here]
```

Comment:
change: 
`see xref:to-configure-munit-maven-plugin-maven.adoc#to-specify-the-runtime-product[here]`
for:
`see xref:to-configure-munit-maven-plugin-maven.adoc#to-specify-the-runtime-product[To Configure MUnit Maven Plugin From the POM File]`